app/reaction.go

```diff
@@ -40,26 +40,31 @@ func (a *App) DeleteReactionForPost(reaction *model.Reaction) *model.AppError {
 		return err
 	}
 
+	hasReactions := true
+	if reactions, _ := a.GetReactionsForPost(post.Id); len(reactions) <= 1 {
```

Comment:
@jwilander This isn't something we need to load test, is it? It's just adding an extra DB query when deleting a reaction, so I don't think it should have a notable performance hit

Reply:
We don't have coverage for deleting reactions in the load test as of now. Using the force I wouldn't think it's any cause for concern